docs/requirements.txt

```diff
@@ -0,0 +1,5 @@
+importlib-metadata==5.2.0
```

Comment:
Shouldn't we also add this line to the ``install_requires`` list in ``setup.py``?

Reply:
I did a few test installs - outside of RtD-builds I could not reproduce the issue. However you are probably right.